combat-trainer.lic

```diff
@@ -1661,6 +1666,34 @@ class TrainerProcess
     game_state.blessed_room = true
   end
 
+  def pray_mat(game_state)
+    waitrt?
+    retreat
+    @equipment_manager.stow_weapon(game_state.weapon_name)
+
+    fput "get #{@prayer_mat} from my #{@theurgy_supply_container}"
+    fix_standing
+    bput("unroll #{@prayer_mat}",
+         'reverently lay your', 'need to be holding that first')
+
+    bput("kneel #{@prayer_mat}", 'You humbly kneel')
+    bput("kiss #{@prayer_mat}", 'You bend forward to kiss')
+
+    fput "get wine from my #{@theurgy_supply_container}"
```

Comment:
Can this (and line 1674) use `bput`?

Reply:
Per your first comment: I've run when out of wine and it's fine (just "What were you referring to" and "Pour what?").  I can update to bput shortly.

Reply:
👌